### PR TITLE
Added to_file option

### PR DESCRIPTION
**Pull request by hryamzik**
Added an option to specify recipients with a file, very useful if you want to send an email to the commit author.

### CODE REVIEW

File: opt/resource/out [hunk 1]
```diff
@@ -54,11 +54,17 @@ def put(src):
...
+def get_recipients(src, payload):
+    recipients = get_value_from_payload(payload, 'params', 'to', optional=True)
+    if not recipients:
+        fname = get_value_from_payload(payload, 'params', 'to_file')
+        with open(os.path.join(src, fname)) as fp:
+            recipients = fp.read().split('\n')
+        if not isinstance(recipients, list):
+            raise ValueError("The recipients in params:to_file have to be a list!")
+    else:
+        if not isinstance(recipients, list):
```
Comment (mdomke) [inline opt/resource/out]: I would prefer to have only one type-check for the `reciepients` list. E.g.:

``` python
recipients = get_value_from_payload(payload, 'params', 'to', optional=True)
if not recipients:
    ...
if not isinstance(recipients, list):
    raise ValueError("The recipients parameter has to be a list!")
```

**Review by mdomke — CHANGES_REQUESTED** [review]
Hey @hryamzik many thanks for your pull-request! The ability to dynamically specify the list of recipients seems to be something people might find useful ;-)

**Comment by hryamzik** [comment]
@mdomke please see the changes.

File: README.md [hunk 2]
```diff
@@ -46,7 +46,7 @@ Sending an email message:
 
 #### Parameters
 
-* `to`: _(mandatory)_ A list of recipients as strings.
+* `to|to_file`: _(mandatory)_ Use `to` to specify a list of recipients as strings or use `to_file` to cpecify file with addresses, each address should be in a new line.
```
Comment (mdomke) [inline README.md]: Hi @hryamzik sorry for the late reply. Would you mind to fix the typo "cpecify" -> "specify" and I'll gladly merge it!

**Review by mdomke — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (hryamzik) [reply README.md]: Indeed, done.

**Comment by mrmcmuffinz** [comment]
Hi, is there an eta when this will go in?

**Comment by hryamzik** [comment]
@mrmcmuffinz feel free to use this image:

```yml
  - name: email-forked
    type: docker-image
    source:
      repository: incaller/concourse-email-resource
```

**Comment by mdomke** [comment]
Sorry, I lost track of this PR. I'll merge it tonight and provide an updated docker image. I'm sorry for the inconvenience.

**Review by mdomke — APPROVED** [review]

**Comment by mdomke** [comment]
@hryamzik, @mrmcmuffinz The updated image is now available on [dockerhub](https://hub.docker.com/r/mdomke/concourse-email-resource/).

**Comment by mrmcmuffinz** [comment]
@mdomke it's not a problem, I was just wondering when the feature was going to come in. Either way thanks to both of you( @hryamzik & @mdomke  )for providing the resource.

**Pull request merged**